phpunit.xml.dist

```diff
@@ -9,17 +9,20 @@
          processIsolation="false"
          stopOnFailure="false"
          syntaxCheck="false"
-         bootstrap="tests/bootstrap.php"
+         bootstrap="vendor/symfony-cmf/testing/bootstrap/bootstrap.php"
 >
     <php>
-        <!-- Disable E_USER_DEPRECATED until 3.0 -->
-        <!-- php -r 'echo -1 & ~E_USER_DEPRECATED;' -->
```

Comment:
ah, i guess that was when symfony started to trigger deprecations. indeed we should not need it anymore. lets see what #486 gives.